aion/lib/aion/room_channel/room.ex

```diff
@@ -50,8 +55,14 @@ defmodule Aion.RoomChannel.Room do
   """
   @spec change_question(%__MODULE__{}, integer) :: __MODULE__.t
   def change_question(room, room_id) do
-    room
-    |> struct(get_new_question_with_answers(room_id))
+    new_questions_with_answers = get_new_question_with_answers(room.questions, room_id)
+    struct(room, new_questions_with_answers)
+  end
+
+  @spec load_questions(%__MODULE__{}, integer) :: __MODULE__.t
+  def load_questions(room, room_id) do
+    questions = room_id |> fetch_questions |> Enum.shuffle
```

Comment:
Use parenthesis for one-arity functions:
https://github.com/christopheradams/elixir_style_guide#parentheses-pipe-operator

Also when we have at least two pipes I would use multiline syntax:
https://github.com/christopheradams/elixir_style_guide#pipe-operator